### ISSUE TITLE
Value interpolation at vertices

### ISSUE THREAD

**Issue by kephale** [cellcanvas/surforama#17]
@alisterburt points out that we should pay attention to the per-vertex interpolation, and possibly consider tricubic.

**Comment by alisterburt** [comment]
not so much the per-vertex interpolation...

https://github.com/cellcanvas/surforama/blob/69a4d75aa2c35deea3fd31b02b9465dc2fceede5/src/surforama/app.py#L187-L188
Above it looks like the density of vertices will affect the resolution of the image being rendered - this is fundamentally quite a big difference to the original membranorama if the mesh vertex density is low

**Comment by kephale** [comment]
Yeah, this is acknowledged. We tested out remeshing before deciding to stick to this approach.

**Comment by kephale** [comment]
I made the issue because even with the difference we should consider per-vertex interpolation. The current method is really just slicing into the array.

**Comment by LorenzLamm** [comment]
Would it make sense to make the interpolation degree optional (as an expert choice or so)? I tried using scipy's map_coordinates function to interpolate the vertex values and ran into quite some efficiency issues with cubic interpolation, so I think this should not be the default option at least.

From left to right: no interpolation --> linear interpolation --> cubic interpolation
<img width="250" alt="surforama_no_interpolation" src="https://github.com/cellcanvas/surforama/assets/34575029/e2e3f488-6b68-4919-a658-2858a44b64f6"><img width="250" alt="surforama_linear_interpolation" src="https://github.com/cellcanvas/surforama/assets/34575029/4cae2d52-1542-4c46-9fa3-f093e64b5627"><img width="250" alt="surforama_cubic_interpolation" src="https://github.com/cellcanvas/surforama/assets/34575029/ad733a05-000e-4daa-bd8e-7934d9bc240d">

This is done with our provided demo example, where at least linear interpolation is necessary. Maybe it makes sense to use this as a default?
